apps/src/studio/blocks.js

```diff
@@ -1502,7 +1502,14 @@ exports.install = function(blockly, blockInstallOptions) {
         this.appendDummyInput()
           .appendTitle(msg.saySprite());
       }
-      if (options.params) {
+      if (options.choices) {
```

Comment:
I might change this param name to something like restrictedChoices or restrictedDialog just to make it more clear.